Modules/vc-module-notifications/VirtoCommerce.NotificationsModule.Core/Model/NotificationParameterAttribute.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace VirtoCommerce.NotificationsModule.Core.Model
+{
+    [AttributeUsage(AttributeTargets.Property)]
```

Comment:
This attribute redundant, we should avoid of using any descriptions at the code level.
For  an available properties  legend  we use JSON resources in UI